packages/2d/src/components/Layout.ts

```diff
@@ -146,15 +148,19 @@ export class Layout extends Node {
   @initial('normal')
   @signal()
   public declare readonly alignItems: SimpleSignal<FlexAlign, this>;
-  @initial(null)
+  @initial(0)
   @signal()
   public declare readonly gap: SimpleSignal<Length, this>;
-  @initial(null)
   @signal()
   public declare readonly rowGap: SimpleSignal<Length, this>;
-  @initial(null)
+  protected getDefaultRowGap() {
+    return this.gap();
```

Comment:
Having both a horizontal and vertical gap makes me think a compound signal would be better here, especially since this will result in:
```tsx
object.rowGap(10);
object.gap(20);
debug(object.rowGap()); // 10
```